project2/styles.css

```diff
@@ -0,0 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Playfair Display', serif;
+    font-size: 17px;
+}
+
+
+.menu li {
+    display: inline-block;
+}
+
+ul {
+    text-align: left;
+    list-style: square;
+    padding-left: 60px;
+    padding-right: 40px;
```

Comment:
Instead of adding padding to each element, consider adding padding rules to your `section` -- it's easier to maintain those consistently than having to check each element you've added unique padding to.